cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -87,12 +126,13 @@ func (m *Manager) Destroy(ctx context.Context, preview *Preview, interactive boo
 func (m *Manager) preview(ctx context.Context, interactive bool) (*PreviewResult, error) {
 	var previewResult *PreviewResult
 
-	previewAndReportProgress := func(spinner *spin.Spinner) error {
+	spinner := spin.NewSpinner("Preparing infrastructure provisioning")
+	err := spinner.Run(func() error {
 		previewTask := m.provider.Preview(ctx)
 
 		go func() {
```

Comment:
Again, should we do this if interactive == false?

Reply:
Spinner issues address and will only ever be created/used in interactive mode.